goosebit/api/v1/devices/routes.py

```diff
@@ -48,4 +48,60 @@ async def devices_delete(_: Request, config: DevicesDeleteRequest) -> StatusResp
     return StatusResponse(success=True)
 
 
+@router.patch(
```

Comment:
This gets added in your first commit 1cb764b1 (message: Add auth token to device database and update migrations). It adds the duplicate for patch and put. Why do we need these two calls? Should not only be the patch code in the commit "Add auth token to API and device patch request" ?